index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header class="header">
+      <nav class="header_nav">
+        <div class="logo black">
+          <a class="black" href="#">Logo</a>
+        </div>
+        <div class="header_menu">
+          <img class="menu_icon" src="assets/ic_menu.png" alt="menu" />
+        </div>
+      </nav>
+    </header>
+    <section class="headline">
+      <h1 class="headline_title">
+        <span class="sub-headline_name">Hey there, I'm Arslan</span>
+        <span class="sub-headline">I'm a software developer</span>
+      </h1>
+      <p class="headline_content">
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don,t hestiate to contact me.
+      </p>
```

Comment:
- [ ] **Required:** Your `h1` headline is wider than the design and your `p` text has a different font than the one defined on the figma file, adjust the sizing and fonts so they matches the figma design

Figma design
![Screenshot from 2023-05-04 14-17-35](https://user-images.githubusercontent.com/50066876/236191012-9f2ce51b-b77b-4d94-abaf-143ab70e750e.png)

Your design
![image](https://user-images.githubusercontent.com/50066876/236191228-f66e2ecf-5bd8-4afb-ae61-d4e53b22995a.png)